### DISCUSSION TITLE
All Data Stopped on 1/11

### DISCUSSION THREAD

**Discussion by mjbigelow** [teslamate-org/teslamate#2394] [General]
Charges, drives, etc. all stopped on 1/11. I got 2021.44.30 on 1/1. 

Has this happened to anyone else? Any thoughts on what I can do? I tried to review the logs but it took forever to generate.

**Comment by cwanja** [comment]
Did you re-login in? I think this was around the time that Tesla pulled a handful of API tokens due to a security issue with exposed TeslaMate installations to the internet. Read more here. https://medium.com/@david_colombo/how-i-got-access-to-25-teslas-around-the-world-by-accident-and-curiosity-8b9ef040a028

And if you have your instance exposed to the internet, upgrade and secure your Grafana instance.

**Comment by mjbigelow** [comment]
Thanks! I rebooted the pi and it works again, so I think this was the
answer. Thank you!!
